tasks/eth/029-holocene-system-config-upgrade-and-init-base/.env

```
@@ -0,0 +1,6 @@
+ETH_RPC_URL="https://c3-chainproxy-eth-base-mainnet-full-dev.cbhq.net"
```

I'm not sure what this rpc url is, can we change it to a public node? 
```suggestion
ETH_RPC_URL="https://ethereum.publicnode.com
```